### DISCUSSION TITLE
Behavior of the selection

### DISCUSSION THREAD

**Discussion by fundon** [toeverything/blocksuite#977] [General]
### Default Page Mode

* [x] Triggers automatic scrolling and selection when upper and lower borders are reached
* [x] Scrolls wheel to select

### Shortcuts

* [ ] `Shift` + `Up`/`Down`

## Block

When to enter block mode?

### Default Page Mode

* [x] Triggers automatic scrolling and selection when upper and lower borders are reached
* [x] Scrolls wheel to select

### Shortcuts

* [ ] `Up`/`Down`
* [ ] `Shift` + `Up`/`Down`

**Comment by himself65** [comment]
maybe related https://github.com/toeverything/blocksuite/pull/988
